features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql

```diff
@@ -1356,6 +1357,21 @@ CREATE TABLE IF NOT EXISTS SCOPE (
     CONSTRAINT SCOPE_UNIQUE UNIQUE (NAME, TENANT_ID)
 )ENGINE INNODB;
 
+CREATE TABLE IF NOT EXISTS APP_ROLE_ASSOCIATION (
+    APP_ID CHAR(36) NOT NULL,
+    ROLE_ID VARCHAR(255) NOT NULL,
+    PRIMARY KEY (APP_ID, ROLE_ID),
+    FOREIGN KEY (APP_ID) REFERENCES SP_APP(UUID) ON DELETE CASCADE
+)DEFAULT CHARACTER SET latin1 ENGINE INNODB;
+
+CREATE TABLE IF NOT EXISTS ROLE_SCOPE (
+    ROLE_ID VARCHAR(255) NOT NULL,
+    SCOPE_NAME VARCHAR(255) NOT NULL,
+    TENANT_ID INT NOT NULL,
+    CONSTRAINT ROLE_SCOPE_UNIQUE UNIQUE (ROLE_ID, SCOPE_NAME, TENANT_ID),
```

Comment:
Shall we convert this to a primary key to avoid issues mentioned in https://github.com/wso2/product-is/issues/12167